### PR TITLE
README: Correct test instructions

### PR DESCRIPTION
**Pull request by felder-cl**
Update the test instructions for building the ethereum contracts.

I may be missing something

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,9 @@ $ ./chainlink
 
 ```bash
 $ yarn
-$ make contracts
+$ (cd belt && yarn setup)
```
Comment (j16r) [inline README.md]: Why are these in parenthesis?

**Comment by j16r** [comment]
I feel like this would be better if we actually had the `contracts` target!

**Comment by felder-cl** [comment]
> I feel like this would be better if we actually had the `contracts` target!

@HenryNguyen5 mentioned there was a one-liner, and he was going to update the instructions, waiting on him ;).

File: README.md (hunk 1 above)
Comment (felder-cl) [reply README.md]: They preserve the callers context.  You don't have to cd back after.

**Pull request merged**